sqlx-core/src/sqlite/options.rs

```diff
@@ -34,17 +36,34 @@ impl FromStr for SqliteConnectOptions {
         let mut options = Self {
             filename: PathBuf::new(),
             in_memory: false,
+            statement_cache_size: 100,
         };
 
         // remove scheme
         s = s
             .trim_start_matches("sqlite://")
             .trim_start_matches("sqlite:");
 
-        if s == ":memory:" {
-            options.in_memory = true;
-        } else {
-            options.filename = s.parse()?;
+        let mut splitted = s.split("?");
```

Comment:
I should be beaten up for this...

Do we want the parameters in the string? I don't like the code I wrote here...

Reply:
Heh.. Ideally we should follow this: https://www.sqlite.org/uri.html

---

If you don't want to dive into that right now it might be best to leave the string parsing alone. Users can always create a `SqliteConnectOptions` manually.

Reply:
I guess I'll just remove these.